docs/topics/modules/planning_development_work.adoc

```diff
@@ -68,36 +34,21 @@ image::gs_feature_eg.png[Example Feature]
 * <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
```

Comment:
I know this is not part of your changes this time, but could we reduce these? Maybe to two tasks so that it's not a lot of administration for a relatively simple Getting Started task.

Reply:
Well Misha the whole point of having this section placed here and all the tasks here is to knit the disparate tasks in the getting started together. This section is what connects all the separate tasks we ask the user to perform. As such I would vote for retaining these tasks.

Reply:
Yeah, this is probably entirely subjective.

I had an idea, though. Instead of just the two of us, we could open this up to a wider audience. I can take this whole "chapter", or at least a part of it, since it's a long chapter, to a minimalism session. Maybe 5.1-5.8 or so.  I can ask the writers discussing it to give us notes about what they suggest would improve the content and note it down and we can implement the changes. It'll be good to get some fresh eyes on the content anyway and I won't say anything about this particular module or anything, just let them look through the basics, flow, text, etc. and give us their thoughts. WDYT?

Reply:
> Yeah, this is probably entirely subjective.

@misha, trying to bring some objectivity to the discussion.
Question 1: what is our intent here?
1. If we want to simply showcase Planner usage, yes 1/2 tasks would be enough
2. If we want to use the Planner section to showcase the feature as well as tie the various seemingly disjoint actions in the Getting started experience together, I believe we need these 4 tasks.
I would suggest lets discuss the intent over a meeting. I am ok with whichever way we decide to go.

Question 2: Do we want to adress the length of the whole module or is it only the Tasks listed here?
If its the length, I have been toying with using the agile template instead of SDD, I have a feeling it could reduce the size a bit. However the number of tasks would be determined by our intent(1st question). If you are ok with this, I will try to have a rough agile draft in place for the meeting so we can evaluate.

On opening docs for minimalism review: +1 I have been thinking of sharing our docs in our minimalism sessions too, we record the sessions so we could simply share the recordings and work on them.
 
Also, please note, since yesterday, design discussions are happening to change the UX flow to make the whole experience cohesive. Should we wait till then? As depending on the changes a lot of things could become redundant. We simply may not need to have so many tasks as the experience may not seem disjoint then. Also, it might be better to have the writers have a look at it in the sessions once we update our docs as per the improved UX. Thoughts?

Reply:
Re: intent - That's why it sticks out so much to me. I feel like the other modules around it are really specific and limited by design so that the user goes through a relatively simple workflow while seeing where to find various features and how to use them. To that end, just having a single work item in the planner to edit the code is probably more than enough, since it illustrates how to use the planner sufficiently as a whole. I think adding multiple work items here complicates it a bit too much when the actual code changes we do later are just copy and pasting code lines or just minor edits. Does that make sense? I think trimming it all down to only the basics for a new user to know would be in line with this document. The details are best left to the user guide.

Q2: As I said, it's just a matter of how much we can look through in about an hour. If there's a lot of feedback, we may only make it through maybe 6-7 modules. If less feedback, we might manage the whole ch 5. I'm open to either. It's important to start from the beginning of the chapter so that the intent and context is present as we go through each module step.

I don't have any issues with the length, please use whichever option you think best fits the workflow, since you're the planner expert. :)

Re: updates. I think there are literally always going to be big updates for a product this young. The minimalism review isn't a judgment on the quality of our docs or how accurate it is to the product, just a bunch of other writers having a first look at something and pointing out bits that could be changed to improve things like the flow of the document, make things obvious if they aren't, enhance the format or structure, etc. Having that sort of feedback is useful at any time so I would recommend we just jump into it as soon as we finish our reviews. It's a bit trying to listen to critique of one's own work so if I submit something, I usually just sit in a corner without argument and listen to the discussion and feedback and note it down. Then, we can discuss the feedback and see how useful it is to us and apply ti accordingly. What do you think?

Reply:
Re: Intent, we have a difference of opinion here, let's hash it out in a meeting. Each of the tasks relate to the actions the user has to perform as part of the getting started.
Re: Length, based on the time I have at hand, I will experiment with the agile template and see if we can arrive at a shorter version.
Re: Minimalism, I understand the limitation of time and sections we can review within the hour. I have been part of these sessions so I know what you mean about writers providing their feedback. I suppose we would be in different minimalism sessions hence it would be good to share the recordings is what I suggested.

Reply:
I'm reluctant to have meetings about differing opinions because they're just that: opinions. It's easier to get a third party's opinion instead and then we have a clear way forward. However, happy to meet to hash this out later. I'll be online later (will message time) and can meet before that time.

Reply:
Agreed, I was suggesting hashing out the intent within the team in our weekly meeting when @rkratky would be available too. Again the group review would add to our input. 
I do not wish to impose on your personal time. Let's discuss in the Tuesday team meeting.

Reply:
Quick notes:

- Agree w/ Misha re the extra WIs. I don't think they help to tie stuff together. Instead, it looks like a homework -- we're not teaching the user how to create a WI, instead, we're telling them "create a whole bunch of them because practice makes perfect". If I were the user, I would snicker and ignore it. We need to be mindful that users don't read our docs because they thirst for knowledge; they read them with disdain when all else failed.
- The whole 5.2.2 section is too long. The list of steps has 8 items and some of them multiple sub-items. Too long.
- Headings should consistently use gerund verb forms.

Reply:
> create a whole bunch of them because practice makes perfect

That was not the intention at all, the intention was to create tasks for each stage of the getting started.
If this is the implication, I will remove the extra WIs. Definitely don't need any snickering and ignoring directed at our docs :)
Re: length, SDD makes the process long. I am working on an Agile workflow, hopefully that would pare it down.